backend/apis/clients/client-interactions/client-interaction-utils.api.js

```diff
@@ -0,0 +1,32 @@
+const mysql = require("mysql");
+const { responseFullName } = require("../../utils/transform-utils");
+
+const modifiableInteractionTypes = [
```

Comment:
Delete this. This is not what determines `canModify`.

Reply:
deleted